karate-tests/src/test/java/patients/healthcareWorker/updatePatientDetails/removeErrors.feature

```diff
@@ -0,0 +1,78 @@
+@sandbox
+Feature: Patch patient - Remove data errors
+
+  # Tests more of the error scenarios that can arise when trying to remove data from a patient resource
+
+  Background:
+    * def utils = call read('classpath:helpers/utils.feature')
+    * def accessToken = karate.callSingle('classpath:patients/healthcareWorker/auth-redirect.feature').accessToken
+    * url baseURL
+
+    * def requestHeaders = call read('classpath:patients/healthcareWorker/healthcare-worker-headers.js')
+    * configure headers = requestHeaders
+    * def nhsNumber = '5900057208'
+    * path 'Patient', nhsNumber
+    * method get
+    * status 200
+    
+    * def patientObject = response
+    * def etag = karate.response.header('etag')
+    * def originalVersion = parseInt(response.meta.versionId)
+
+  Scenario: Error: attempt to remove name object from patient without prior test
+    * def diagnostics = "Invalid update with error - removal '/name/1' is not immediately preceded by equivalent test - instead it is the first item"
+    * def expectedBody = read('classpath:mocks/stubs/errorResponses/INVALID_UPDATE.json')
+
+    * def requestHeaders = call read('classpath:patients/healthcareWorker/healthcare-worker-headers.js')
+    * configure headers = requestHeaders 
+    * header Content-Type = "application/json-patch+json"
+    * header If-Match = etag
+    * path 'Patient', nhsNumber
+    * request {"patches":[{"op":"remove","path":"/name/1"}]}
```

Comment:
name/1 doesn't exist in the first place at this point. Although I understand that's not the point of this particular test.